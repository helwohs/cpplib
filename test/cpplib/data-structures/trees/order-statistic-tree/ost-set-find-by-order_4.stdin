10
1 8 7 1 3 2 2 7 9 9
7 7 4 1 4 2 1 1 2 1
