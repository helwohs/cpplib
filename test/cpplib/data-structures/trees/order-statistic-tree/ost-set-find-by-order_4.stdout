2 7 1 1 2 7 8 3 9 9 
